intermol/tests/README.md

```diff
@@ -0,0 +1,32 @@
+### Running the tests
+In order to cover the large number of functional forms for different energy
+terms, we use a testing suite that converts unit tests, e.g. a tiny system with
+harmonic bonds between, and stress tests, e.g., a large solvated protein or
+micelle, between Gromacs, Lammps and Desmond.
+
+To run ALL tests (this may take a while), use:
+```bash
```

Comment:
Yes it does. I've turned off the stress tests when running on CI but we could add a flag too.